flow-typed/npm/jest_v23.x.x.js

```diff
@@ -1,5 +1,6 @@
 // flow-typed signature: f5a484315a3dea13d273645306e4076a
 // flow-typed version: 7c5d14b3d4/jest_v23.x.x/flow_>=v0.39.x
```

Comment:
#426 updates jest to 24 which includes a new version of this file with the updated jest-enzyme matchers.